src/backend/Csrs.Api/Authentication/AuthenticationExtensions.cs

```diff
@@ -35,9 +35,9 @@ public static void AddJwtBearerAuthentication(this WebApplicationBuilder builder
             options.MetadataAddress = options.GetMetadataAddress();
 
             //For docker keycloak which does not use ssl
-            if  (options.GetMetadataAddress().Contains("http://")) { 
-                options.RequireHttpsMetadata = false;
-            }
+            //if  (options.GetMetadataAddress().Contains("http://")) { 
```

Comment:
remove, just dont comment out. This can be handled by configuring Jwt:RequireHttpsMetadata  = false

Reply:
done

Reply:
Please adjust it as per Phil's comment